python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sys
+import cPickle
+import os
+
+def execute(config, project, name, entities, base_configuration, engine_name):
+    import sgtk
+
```

Comment:
i suggest init logging to `tk-desktop` right away up here. 
http://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.log.LogManager.initialize_base_file_handler